ontologies.ofn

```diff
@@ -20,7 +20,9 @@ Import(<http://purl.obolibrary.org/obo/eco/eco-base.owl>)
 Import(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-caro.owl>)
 Import(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-caro.owl>)
 Import(<http://purl.obolibrary.org/obo/go/extensions/go-bfo-bridge.owl>)
-Import(<http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl>)
+# Commented out on 2023-jun-19 by Gaurav: appears to be redundant with
+# http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl
+# Import(<http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl>)
```

Comment:
This can be reverted—NCBITaxon is fixed.

Reply:
Reverted in ed85f7b.